components/navigationRail.tsx

```
@@ -0,0 +1,142 @@
+              {mailboxes?.map((mailbox) => (
+                <DropdownMenuItem
+                  key={mailbox.slug}
+                  onClick={() => {
+                    setMailboxSlug(mailbox.slug || "");
+                    router.push(`/mailboxes/${mailbox.slug}/conversations`);
+                  }}
+                  className="flex items-center gap-2"
+                >
+                  <Avatar src={undefined} fallback={mailbox.name} size="sm" />
+                  <span className="truncate text-base">{mailbox.name}</span>
+                  <span className="ml-auto">{mailbox.slug === currentMailbox?.slug && <CheckCircle className="text-foreground w-4 h-4" />}</span>
```

ESLint

[error] Replace `{mailbox.slug·===·currentMailbox?.slug·&&·<CheckCircle·className="text-foreground·w-4·h-4"·/>}` with `⏎····················{mailbox.slug·===·currentMailbox?.slug·&&·<CheckCircle·className="text-foreground·w-4·h-4"·/>}⏎··················`

(prettier/prettier)